tests/p2p/test_tx_pool.py

```diff
@@ -0,0 +1,154 @@
+import asyncio
+import pytest
+import random
+
+from p2p.peer import (
+    ETHPeer,
+)
+from p2p.eth import (
+    Transactions
+)
+from evm.rlp.transactions import (
+    BaseTransactionFields
+)
+
+from trinity.tx_pool.pool import (
+    TxPool,
+)
+
+from tests.p2p.peer_helpers import (
+    get_directly_linked_peers,
+    MockPeerPoolWithConnectedPeers,
+)
+
+# TODO: This whole file should live in the trinity tests but can't
```

Comment:
@pipermerriam any advice how to deal with that? My limited Python fu tells me the problem is that most of the directories within `tests` aren't actually Python modules (no `__init__.py`) and that is the reason why I can't cross import it (`..relative.lookup` also doesn't work).

But I read somewhere that NOT making the test directories proper python modules is actually the right way to handle test directories because of some pytest madness. Any advice how to handle this?

Reply:
Maybe relocating `peer_helpers` to something like `p2p.utils.testing.peer_helpers`.  That will cause it to be shipped with the actual `p2p` module but I don't see that as a big deal as long as the overhead remains small (like not including large fixture files or something like that).

Reply:
Mmh...not terribly excited about that :sweat_smile: I digged a bit further and noticed that `tests` already had a `__init__.py` but `tests/p2p` did not. So, I tried what would happen if we make that a module and this seems to work: https://github.com/ethereum/py-evm/pull/898/commits/882780972fae53777b5e129d1f23b51851a6563d

Still has some things failing but I think that could be worked out. But maybe we leave this out of this PR and I address that in a follow up?

Reply:
I slightly changed that comment and plan to address that test refactoring in a follow up PR to not grow this PR with more unrelated changes.